styles/styles.css

```diff
@@ -33,6 +33,11 @@
   --body-font-size-xs: 12px;
   --body-font-size-s: 14px;
   --body-font-size-m: 16px;
+  --body-font-size-l: 18px;
```

Comment:
These font sizes are just for the preview, I am fully expecting to wait and refactor after the global styles are in `main`.